dd-sdk-android/src/main/kotlin/com/datadog/android/rum/internal/domain/scope/RumActionScope.kt

```diff
@@ -201,6 +203,7 @@ internal class RumActionScope(
                 id = context.sessionId,
                 type = ActionEvent.ActionEventSessionType.USER
             ),
+            source = rumEventSourceResolver.resolveActionEventSource(CoreFeature.sourceName),
```

Comment:
One more thing, I guess I should have seen it before, but you're running this "resolveXxxEventSource" for every single event, but the `CoreFeature.sourceName` is immutable. Wouldn't it be better to resolve it once and for all (say when the `RumFeature` is initialized)? That would save a lot of useless operations, wdyt?

Reply:
I also thought about this...then I think the operation is not very costly and keeping those values at the RumFeature level will look weird, I could keep those at the scope level though.